packages/react-ui/__tests__/widgets/BarWidgetUI.test.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import BarWidgetUI from '../../src/widgets/BarWidgetUI';
+import { getMaterialUIContext, mockEcharts } from './testUtils';
+
+describe('BarWidgetUI', () => {
+  beforeAll(() => {
+    mockEcharts.init();
+  });
+
+  afterAll(() => {
+    mockEcharts.destroy();
+  });
+
+  const Y_AXIS_DATA = [1, 2, 3];
+
+  const X_AXIS_DATA = ['column_1', 'column_2', 'column_3'];
+
+  const Widget = (props) =>
+    getMaterialUIContext(
```

Comment:
This is perfectly fine. If you check cloud-native, we have a custom render, which automatically adds this (and other contexts as required). That's a nice, recommended pattern here: https://testing-library.com/docs/react-testing-library/setup/, in case you find it interesting to use